lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -371,14 +374,46 @@ public void indexObjects(
                 pw.println(UObject.transformObjectToString(doc));
             }
             pw.close();
-            makeRequestBulk("POST", indexName, tempFile);
+            Response response = makeRequestBulk("POST", indexName, tempFile);
+
+            verify(response);
+
             updateLastVersionsInData(indexName, pguid, lastVersion);
         } finally {
             tempFile.delete();
         }
         refreshIndex(indexName);
     }
-    
+
+
+    private void verify(Response response) throws IOException {
+        // convert JSON string to Map
+        String responseJsonStr = EntityUtils.toString(response.getEntity());
+
+        ObjectMapper mapper = new ObjectMapper();
+        final Map<String, Object> responseMap = mapper.readValue(responseJsonStr,
+                new TypeReference<Map<String, Object>>() {});
+
+        // check for and collect response data from index operations that failed
+        List errorItems = new ArrayList();
+        if(responseMap.get("errors").equals(Boolean.TRUE)) {
```

Comment:
space to distinguish key words from functions all through this part

Reply:
Fixed.